kie-ci/src/main/java/org/kie/scanner/MavenRepository.java

```
@@ -188,32 +191,38 @@ public Version resolveVersion( String artifactName ) {
     public void deployArtifact( ReleaseId releaseId,
                                 InternalKieModule kieModule,
                                 File pomfile ) {
-        File jarFile = bytesToFile( releaseId, kieModule.getBytes(), ".jar" );
-        deployArtifact( releaseId, jarFile, pomfile );
+        RemoteRepository repository = getRemoteRepositoryFromDistributionManagement( pomfile );
+        if (repository == null) {
```

In this overloaded version of the deployArtifact method no RemoteRepository is passed, so I assume that I should take that repo from the distribution management. If also there's no distribution management I don't know where the artifact should be deployed so I just throw that Exception. Should I just silently ignore this issue and don't perform any deploy? Should I log a warning message?